src/MeshInertiaCalculator.hh

```diff
@@ -68,11 +68,21 @@ namespace gz
       /// The calculation method used in this class is described here:
       /// https://www.geometrictools.com/Documentation/PolyhedralMassProperties.pdf
       /// and it works on triangle water-tight meshes for simple polyhedron
-      class MeshInertiaCalculator
+      class GZ_SIM_VISIBLE MeshInertiaCalculator
       {
         /// \brief Constructor
         public: MeshInertiaCalculator() = default;
 
+        /// \brief Function to correct an invalid mass matrix. The mass matrix
+        /// to be corrected needs to be positive definite and within a small
+        /// tolerance of satisfying the triangle inequality test. If the above
+        /// conditions are not satisfied, the mass matrix will not be corrected.
+        /// \param[in, out] _massMatrix Mass matrix to correct
+        /// \return True if the mass matrix is corrected, false otherwise.
+        public: static bool CorrectMassMatrix(
+            gz::math::MassMatrix3d &_massMatrix,
+            double tol = kPrincipalMomentPercentTol);
```

Comment:
please document the `tol` parameter in doxygen

Reply:
[a7ba9bf](https://github.com/gazebosim/gz-sim/pull/2775/commits/a7ba9bf7baf4baef65802d3a9b6f0931449a84b5)